Prog/Updates/Update_Sec8mf_2022_12.sas

```diff
@@ -0,0 +1,25 @@
+/**************************************************************************
+ Program:  Update_Sec8mf_2022_12.sas
+ Library:  PresCat
+ Project:  Urban-Greater DC
+ Author:   Donovan Harvey
+ Created:  1/25/2023
+ Version:  SAS 9.4
+ Environment:  Local Windows session (desktop)
+ GitHub issue:  Update-#332
+ 
+ Description:  Update Preservation Catalog with latest HUD Sec 8 MF
+ update file.
+
+ Modifications:
+**************************************************************************/
+
+%include "\\sas1\DCdata\SAS\Inc\StdLocal.sas";
+
+** Define libraries **;
+%DCData_lib( PresCat )
+%DCData_lib( HUD )
+
+options nominoperator;
+%Update_Sec8mf( Update_file=Sec8mf_2022_12 )
```

Comment:
This looks good but there is a Section 8 project that appears not to be in our Catalog. Strange. I will create a separate issue to add it.